jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import tempfile
+from zipfile import ZipFile
+import pathlib
+
+from astropy.utils.data import download_file
+
+from jdaviz.configs.mosviz.helper import MosViz
+
+@pytest.fixture
+def mosviz_helper():
+    return MosViz()
+
+@pytest.mark.remote_data
+def test_niriss_parser(mosviz_helper):
+    data_dir = tempfile.gettempdir()
+
+    test_data = 'https://stsci.box.com/shared/static/9lkf5zha6zkf8ujnairy6krobbh038wt.zip'
```

Comment:
What is the size of this when uncompressed? Want to make sure we don't hit the CI quota.

If it is too big, we have to consider generating smaller set of fake data on the fly or using smaller download data set.

Reply:
Good call, that zip file contains a bunch of files not needed for this. I can upload a new zip file with only the files we need, which comes out to ~100MB. Is that big enough to cause problems?

Reply:
That should be ok

Reply:
Alright, I'll leave the remaining files as-is for now then, I'll update the link shortly.

Reply:
Yes, 100 MB should be okay unless it decompressed into like 500 MB worth of files. 😉

Reply:
Nah, 100 MB is the decompressed size.